### PR TITLE
feat(OHI): Added new docs for NGINX and mySQL

### PR DESCRIPTION
**Pull request by x8a**

### CODE REVIEW

File: src/nav/infrastructure.yml [hunk 1]
```diff
@@ -618,13 +618,21 @@ pages:
           - title: Microsoft SQL Server integration
             path: /docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql-server-monitoring-integration
           - title: MySQL integration
-            path: /docs/infrastructure/host-integrations/host-integrations-list/mysql-monitoring-integration
+            pages:
+              - title: Installing the MySQL integration
+                path: /docs/infrastructure/host-integrations/host-integrations-list/mySQL/mysql-integration
+              - title: MySQL integration's advanced configuration
```
Comment (urbiz-nr) [inline src/nav/infrastructure.yml:624]: I'd call this one "MySQL advanced config" for the short, left panel title

File: src/nav/infrastructure.yml [hunk 2]
```diff
@@ -618,13 +618,21 @@ pages:
...
             path: /docs/infrastructure/host-integrations/host-integrations-list/nagios-monitoring-integration
           - title: NFS integration
             path: /docs/infrastructure/host-integrations/host-integrations-list/nfs-monitoring-integration
           - title: NGINX integration
-            path: /docs/infrastructure/host-integrations/host-integrations-list/nginx-monitoring-integration
+            pages:
+              - title: Installing the NGINX integration
+                path: /docs/infrastructure/host-integrations/host-integrations-list/nginx/nginx-integration
+              - title: NGINX integration's advanced configuration
+                path: /docs/infrastructure/host-integrations/host-integrations-list/nginx/nginx-advanced-config
```
Comment (urbiz-nr) [inline src/nav/infrastructure.yml:635]: Same, I'd call this one "NGNIX advanced config" for the short, left panel title

Comment (urbiz-nr) [reply src/nav/infrastructure.yml:635]: And "Install NGNIX"

File: src/nav/infrastructure.yml (hunk 1 above)
Comment (urbiz-nr) [reply src/nav/infrastructure.yml:624]: And "Install MySQL"

Comment (x8a) [reply src/nav/infrastructure.yml]: Thanks! 👏🏻

**Review by urbiz-nr — DISMISSED** [review]
Tried locally, all looks good. Nice job!

**Comment by roadlittledawn** [comment]
curious why we edited the i18n file:
```src/i18n/content/jp/docs/infrastructure/infrastructure-integrations/get-started/understand-use-data-infrastructure-integrations.mdx```

**Comment by x8a** [comment]
@roadlittledawn, I edited a link in that file. It's my understanding that we can edit links and other things not related to translatable content. Is that not true anymore?

**Pull request merged**